.github/workflows/ci-cd.yml

```diff
@@ -155,10 +155,10 @@ jobs:
             no-extensions: 'Y'
             os: ubuntu
             experimental: false
-          # - os: ubuntu
-          #   pyver: "3.14"
-          #   experimental: true
-          #   no-extensions: 'Y'
+          - os: ubuntu
+            pyver: "3.14.0-alpha.2"
```

Comment:
Leave it as 3.14 as before and it will just take the latest release. I don't think we want to update this every month..